src/System.Runtime.Extensions/src/System/Runtime/CompilerServices/SwitchExpressionException.cs

```diff
@@ -31,12 +31,12 @@ private SwitchExpressionException(SerializationInfo info, StreamingContext conte
             UnmatchedValue = info.GetValue(nameof(UnmatchedValue), typeof(object));
         }
 
-        public SwitchExpressionException(string message) : base(message) { }
+        public SwitchExpressionException(string? message) : base(message) { }
 
-        public SwitchExpressionException(string message, Exception innerException)
+        public SwitchExpressionException(string? message, Exception? innerException)
             : base(message, innerException) { }
 
-        public object UnmatchedValue { get; }
```

Comment:
should `public override string Message` below be nullable?

Reply:
No, it'll never return null.